### ISSUE TITLE
order of Hebrew letters in correction of former Latin-lettered text

### ISSUE THREAD

**Issue by mirjam-amsterdam** [urieli/jochre#65]
might be related to #58
nybc208621 says in the publisher-field: s.l.
perhaps that is anyways a case that should not be fiddled with by users.
it should probably say s.l. : s.n.
I tried to correct s.l. into London, in Hebrew characters. Only, apparently the field is set as left to right because of the s.l. ?
the Hebrew letters wrote themselves from left to right, no matter if I used the on-screen-keyboard, or my own laptop-keyboard, 
See screenshot for the on-screen.
![london - s l](https://user-images.githubusercontent.com/44893880/92460050-c68aea00-f1c7-11ea-8347-26cf90ccca90.JPG)

**Comment by urieli** [comment]
No, the field is set "left-to-right" by default, it has nothing to do with the existing content.

What we really need here is two separate fields, just like for Title and Author: one containing Latin characters (either transcription or the Latin-character original of the publisher's name), and one containing Hebrew characters. I'll bring this up with the Yiddish Book Center.
